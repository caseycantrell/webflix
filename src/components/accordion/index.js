import { useState } from "react";

export default function Accordion({ children, ...restProps }){
    return (
        <Container {...restProps}>
            <Inner>
                {children}
            </Inner>
        </Container>
    )
};

Accordion.Title = function AccordionTitle({ children, ...restProps }) {
    return <Title {...restProps}></Title>
};

Accordion.Frame = function AccordionFrame({ children, ...restProps }) {
    return <Frame {...restProps}></Frame>
};

Accordion.Item = function AccordionItem({ children, ...restProps }) {
    const [toggleShow, setToggleShow] = useState(false);
    return <Item {...restProps}></Item>
};

Accordion.Header = function AccordionHeader({ children, ...restProps }) {
    return <Header {...restProps}></Header>
};