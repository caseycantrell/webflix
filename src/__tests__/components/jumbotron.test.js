import { render } from "@testing-library/react";
import { Jumbotron } from "../../components";
import jumboData from "../../fixtures/jumbo.json";