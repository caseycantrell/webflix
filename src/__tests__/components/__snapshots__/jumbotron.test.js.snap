// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Jumbotron /> renders ye old <Jumbotron /> with data 1`] = `
<div
  class="sc-gKclnd hXogEG"
>
  <div
    class="sc-bdvvtL cGjNFb"
  >
    <div
      class="sc-gsDKAQ crzAFp"
      direction="row"
    >
      <div
        class="sc-dkPtRN bYePld"
      >
        <h1
          class="sc-hKwDye gnjbXq"
        >
          Enjoy on your TV.
        </h1>
        <h2
          class="sc-eCImPb foLRQI"
        >
          Watch on Smart TVs, Playstation, Xbox, Chromecast, Apple TV, Blu-ray players, and more.
        </h2>
      </div>
      <div
        class="sc-dkPtRN bYePld"
      >
        <img
          alt="Tiger King on Webflix"
          class="sc-jRQBWg dLnrgV"
          data-testid="1-jumbo-image"
          src="/images/misc/home-tv.jpg"
        />
      </div>
    </div>
  </div>
  <div
    class="sc-bdvvtL cGjNFb"
  >
    <div
      class="sc-gsDKAQ nqoVQ"
      direction="row-reverse"
    >
      <div
        class="sc-dkPtRN bYePld"
      >
        <h1
          class="sc-hKwDye gnjbXq"
        >
          Download your shows to watch offline.
        </h1>
        <h2
          class="sc-eCImPb foLRQI"
        >
          Save favorites easily and always have something to watch.
        </h2>
      </div>
      <div
        class="sc-dkPtRN bYePld"
      >
        <img
          alt="Watch on mobile"
          class="sc-jRQBWg dLnrgV"
          data-testid="2-jumbo-image"
          src="/images/misc/home-mobile.jpg"
        />
      </div>
    </div>
  </div>
  <div
    class="sc-bdvvtL cGjNFb"
  >
    <div
      class="sc-gsDKAQ crzAFp"
      direction="row"
    >
      <div
        class="sc-dkPtRN bYePld"
      >
        <h1
          class="sc-hKwDye gnjbXq"
        >
          Watch everywhere.
        </h1>
        <h2
          class="sc-eCImPb foLRQI"
        >
          Stream unlimited movies and TV shows on your phone, tablet, laptop and TV without paying more.
        </h2>
      </div>
      <div
        class="sc-dkPtRN bYePld"
      >
        <img
          alt="Money Heist on Webflix"
          class="sc-jRQBWg dLnrgV"
          data-testid="3-jumbo-image"
          src="/images/misc/home-imac.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
